### ISSUE TITLE
Brick API missing

### ISSUE THREAD

**Issue by vinsaTimeparity** [paymentwall/paymentwall-dotnet#3]
Hi team,
       The Brick API is not availabe in the .Net API ? Could you please suggest if it will be integrated in the nuget package anytime soon ? @ivan-kovalyov

**Comment by nkovacic** [comment]
Yep I also noticed that brick API is missing. Is this feature on the roadmap?
Would shorten the integration time considerably.
